### ISSUE TITLE
Batch size vs rank 1

### ISSUE THREAD

**Issue by cptay** [Cysu/open-reid#28]
Hi,

I have 3 questions here

1) Using the code provided here, I tested the market 1501 dataset using the following batch sizes (My 8Gb GPU card does not allow higher batch size)

batch size     ( Rank1 )
    64              ( 77.4% )
    96              ( 79.5% )
   108             ( 79.8% )

Is this the expected results? Running a batch size of 256 will get me ~84% rank 1?

2) Also, there seems no different in the result whether image normalization is used or not. True?

3) I have memory leakage problem. Maximum epoch run before OS terminate the program is about 18-20, depending on batch size. Not sure if this is a problem of running Pytorch under Windows 10, or python code issues?

Thanks

**Comment by zydou** [comment]
Hi, @cptay 

1. In my experiments, running a batch size of 256 gives me ~67.9% mAP and ~84% rank 1. Larger batch size will get better results.

2. I'm not sure about it.

3. Pytorch is not officially supported on Windows. The code running on Linux works perfectly.

**Comment by cptay** [comment]
Hi Zhiyong, Thanks for the prompt reply.

**Comment by Cysu** [comment]
@zydou Thanks very much for your answer!

@cptay 
1. Larger batch size works better for triplet loss, since you can find harder negative samples.
2. The effective of using normalization could be very minor.
3. Sorry but I have no experience with PyTorch on Windows.

**Comment by cptay** [comment]
Hi,

May I know if you used test-time augmentation during evaluation?

Thanks

**Comment by Cysu** [comment]
@cptay No we don't have data augmentation for test evaluation. Just resizing and normalizing the whole image at [here](https://github.com/Cysu/open-reid/blob/master/examples/softmax_loss.py#L43-L47).
